test/dao/operational.ts

```diff
@@ -0,0 +1,27 @@
+// Declare imports
+import * as helpers from '../helpers/contract-helpers';
+import { expect } from 'chai';
+
+let ssvNetworkContract: any;
```

Comment:
Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)